stylesheets/styles.css

```diff
@@ -0,0 +1,112 @@
+.div {
+  width: 800px;
+  height: 600px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+}
```

Comment:
I don't see any tags with a class of div (a more descriptive name would be helpful too). So it doesn't look like this rule actually gets applied, and all of the grid-related rules for the header, etc, essentially have no effect. Your use of flex for the two sections of projects worked great, but try to take another swing at using grids other projects or your capstone.